### PR TITLE
add mca collection

### PR DESCRIPTION
**Pull request by yhpoh91**
## Context

This is for adding a new MCA collection and environment that can be shared with partners usage.

## Checklist
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: multi-currency-account/MCA Sandbox.postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{
+	"id": "ed4c11ff-653f-4f92-abfc-50548c2f73fa",
+	"name": "MCA Sandbox",
+	"values": [
+		{
+			"key": "client-id",
+			"value": "tw-test-bank",
```
Comment (JamesFerretti) [inline multi-currency-account/MCA Sandbox.postman_environment.json:7]: We probably don't want these to be public?

File: multi-currency-account/Multi-Currency Account.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,1687 @@
...
+									"type": "text"
+								},
+								{
+									"key": "client_id",
+									"value": "{{client-id}}",
+									"type": "text"
+								},
+								{
+									"key": "code",
+									"value": "JjnyiC",
```
Comment (JamesFerretti) [inline multi-currency-account/Multi-Currency Account.postman_collection.json:261]: Remove

File: multi-currency-account/Multi-Currency Account.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,1687 @@
...
+									"value": "{{token}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"type\": \"personal\",\n    \"details\" : {\n        \"firstName\": \"Test\",\n        \"lastName\": \"Profile\",\n        \"dateOfBirth\": \"1990-01-01\",\n        \"phoneNumber\": \"+6581234567\"\n    }\n}",
```
Comment (JamesFerretti) [inline multi-currency-account/Multi-Currency Account.postman_collection.json:313]: +6581234567
Is this a random number? maybe good to make it more generic?

**Review by JamesFerretti — CHANGES_REQUESTED** [review]

File: multi-currency-account/Multi-Currency Account.postman_collection.json (hunk 3 above)
Comment (yhpoh91) [reply multi-currency-account/Multi-Currency Account.postman_collection.json:313]: this is a random number that is valid for Singapore mobile number format

**Review by JamesFerretti — APPROVED** [review]

**Pull request merged**